packages/@aws-cdk/aws-servicecatalog/lib/private/util.ts

```diff
@@ -7,4 +7,4 @@ export function hashValues(...ids: string[]): string {
   const sha256 = crypto.createHash('sha256');
   ids.forEach(val => sha256.update(val));
   return sha256.digest('hex').slice(0, 12);
-}
+}
```

Comment:
Why...?

Reply:
not sure what/how this snuck in.

Reply:
Still here 😉.

Reply:
didn't see what change was but I think this is removed now

Reply:
nvm